app/views/local_transaction/unavailable_service.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render layout: 'shared/base_page', locals: {
+  title: "This service is not available in #{@country_name}",
+  publication: @publication,
+  edition: @edition,
+} do %>
+  <div class="interaction">
+    <p class="govuk-body">
+      We’ve matched the postcode <span class="postcode"><%= @postcode %></span> to <span class="local-authority"><%= @local_authority.name %></span>.
+    </p>
+
+    <p class="govuk-body">
+      This service is not available in <span class="country-name"><%= @country_name %></span>. You can find other services on the <span class="local-authority"><%= @local_authority.name %></span> website.
+    </p>
+
+    <p id="get-started" class="get-started group">
+      <%= render "govuk_publishing_components/components/button", {
+        text: "Find other services",
+        rel: "external",
+        start: true,
+        margin_bottom: true,
+        href: @interaction_details["local_interaction"]["url"],
+      } %>
+    </p>
+  </div>
+
+  <div class="search-again">
+    <%= render "govuk_publishing_components/components/back_link", {
+      href: local_transaction_search_path(@publication.slug),
+    } %>
+  </div>
```

Comment:
Think this should be above the body content on line 6

Reply:
This change would also be required in `https://github.com/alphagov/frontend/blob/master/app/views/local_transaction/results.html.erb` for consistency.  

Are you happy for me to do this as a separate commit as part of this PR, or would you prefer this change to be a completely separate story/PR?

This also applies to https://github.com/alphagov/frontend/pull/2550#discussion_r543416633

Reply:
I don't think we are dependent on the results page being correct and we don't need to fix results page as part of the PR. As this is an entirely new page it probably should be correct from the start. It'd be better that we keep consistency with the rest of GOV.UK, rather than consistency with the results page thats going to be fixed anyway.

Reply:
Moved on `unavailable_service` page only